diff --git a/pango/modules.c b/pango/modules.c
index ca8d488..cd00eba 100644
--- a/pango/modules.c
+++ b/pango/modules.c
@@ -450,6 +450,7 @@ process_module_file (FILE *module_file, const gchar *module_file_dir)
 	  switch (i)
 	    {
 	    case 0:
+#if 0
 	      if (!g_path_is_absolute (tmp_buf->str)
 #ifdef __APPLE__
 	          && strncmp (tmp_buf->str, "@executable_path/", 17)
@@ -467,7 +468,9 @@ process_module_file (FILE *module_file, const gchar *module_file_dir)
 		  g_string_assign (tmp_buf, abs_file_name);
 		  g_free ((gpointer) abs_file_name);
 		}
+#endif
 	      pair->module = find_or_create_module (tmp_buf->str);
+
 	      break;
 	    case 1:
 	      pair->info.id = g_strdup (tmp_buf->str);
