--- cairo-1.12.16-original/Makefile.am	2013-08-27 01:11:51.000000000 +0900
+++ cairo-1.12.16/Makefile.am	2013-12-13 19:34:18.206462845 +0900
@@ -16,11 +16,11 @@ EXTRA_DIST += \
 
 ACLOCAL_AMFLAGS = -I build ${ACLOCAL_FLAGS}
 
-DIST_SUBDIRS = src doc util boilerplate test perf
+DIST_SUBDIRS = src doc util boilerplate perf
 SUBDIRS = src doc util
 # libpng is required for our test programs
 if CAIRO_HAS_PNG_FUNCTIONS
-SUBDIRS += boilerplate test perf
+SUBDIRS += boilerplate perf
 endif
 
 configure: cairo-version.h
