diff -up cairo-1.12.16-orig/build/configure.ac.warnings cairo-1.12.16/build/configure.ac.warnings
--- cairo-1.12.16-orig/build/configure.ac.warnings	2013-08-27 00:07:21.000000000 +0900
+++ cairo-1.12.16/build/configure.ac.warnings	2013-12-17 17:30:15.422665322 +0900
@@ -49,7 +49,10 @@ AC_TRY_LINK([],[
 MAYBE_WARN="$MAYBE_WARN -fno-strict-aliasing -fno-common"
 
 dnl Also to turn various gcc/glibc-specific preprocessor checks
-MAYBE_WARN="$MAYBE_WARN -Wp,-D_FORTIFY_SOURCE=2"
+dnl
+dnl XXX Disable this FORTIFY enableing for the purpose of targetting
+dnl old versions of glibc
+dnl MAYBE_WARN="$MAYBE_WARN -Wp,-D_FORTIFY_SOURCE=2"
 
 # invalidate cached value if MAYBE_WARN has changed
 if test "x$cairo_cv_warn_maybe" != "x$MAYBE_WARN"; then
